lib/checkSystem.spec.js

```diff
@@ -88,7 +87,7 @@ test('checkSystem', (t) => {
 
         const checkSystem = proxyquire('./checkSystem', {
             child_process : {
-                exec: (command, cb) => {
+                exec: (_, cb) => {
```

Comment:
Please change this back to `'command'`.  I think it makes it more clear the API from Node we are replacing

Reply:
@mohlsen I used xo linter and fulfilled its recommendations. Unused variables must be **_**